source/Octopus.Client/Repositories/Async/PackageMetadataRepository.cs

```diff
@@ -15,8 +15,8 @@ public PackageMetadataRepository(IOctopusAsyncRepository repository)
 
         public async Task<OctopusPackageMetadataMappedResource> Get(string id)
         {
-            var rootDocument = await repository.Client.Repository.LoadRootDocument();
```

Comment:
The root document's links will contain fully qualified links for the default space, for backwards compatibility. If the default space is disabled, then the root document won't even contain these links.